authzforce-xacmlsdk-samples/src/main/java/org/ow2/authzforce/sdk/pap/SimplePolicyCRUD.java

```diff
@@ -0,0 +1,74 @@
+package org.ow2.authzforce.sdk.pap;
+
+
+import oasis.names.tc.xacml._3_0.core.schema.wd_17.PolicySet;
+import org.ow2.authzforce.rest.api.xmlns.DomainProperties;
+import org.ow2.authzforce.sdk.AdminXacmlSdk;
+import org.ow2.authzforce.sdk.impl.AdminXacmlSdkImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3._2005.atom.Link;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class SimplePolicyCRUD {
+    private static final Logger log = LoggerFactory.getLogger(SimplePolicyCRUD.class);
+
+    private static final String ENDPOINT_ADDRESS = "http://127.0.0.1:8080/authzforce-ce";
+    private static final String DOMAIN = "myTestDomain1";
+
+    public static void main(String[] args) throws Exception {
+        AdminXacmlSdk client = new AdminXacmlSdkImpl(URI.create(ENDPOINT_ADDRESS));
+
+        log.warn("***********domain CRUD operations***********");
+        log.error("(C)reate");
+        List<Link> domains = client.getDomains();
+        log.info("{} domains found: {}", domains.size(), domains.stream().map(Link::getHref).toArray());
+        String newDomainID = client.addDomain("test domain", DOMAIN);
+
+        log.error("(R)ead");
+        domains = client.getDomains();
+        log.info("{} domains found: {}", domains.size(), domains.stream().map(Link::getHref).toArray());
+
+        DomainProperties domainProperties = client.getDomain(newDomainID);
+        log.info("Friendly domain name: {}", domainProperties.getExternalId());
+
+        log.error("(U)pdate");
+        client.addDomain("test domain", DOMAIN);
+
+        log.error("(D)elete... lets leave it till the end, if we delete the domain now we cant to policy CRUD on it");
```

Comment:
log.error("(D)elete... lets leave it till the end, if we delete the domain now we cant do policy CRUD on it");

Reply:
should be a typo

Reply:
Done